kernel/config/definitions/kernel.yml

```diff
@@ -80,9 +83,7 @@ If this is checked, then the binaries will be built and pushed to chacra even if
 
     scm:
```

Comment:
you don't want ``scm`` in this job at all because that will make this job respond to pushes to ceph-client.git as well.